stoplight.js

```diff
@@ -0,0 +1,31 @@
+//For some reason my "spotlight.js" wouldn't link
+//to the page, tried changing the tag and their position
+//please halp!
+
+$( document ).ready(function() {
+    console.log( "ready!" );
+});
+if ($("#red").on(click, function())) {
+  $("#red").addClass("active");
+  $("#red").removeClass("active");
+}
+else
+{$("#red").addClass("active")};
+});
+
+if ($("#yellow").on(click, function())) {
+  $("#yellow").addClass("active");
+  $("#yellow").removeClass("active");
+}
+else {
+  $("#yellow").addClass("active");
+}
+});
+
+if ($("#green").on(click, function())) {
+  $("#green").addClass("active");
+  $("#green").removeClass("active");
+  else {
+    $("#green").addClass("active");
+  }
+});
```

Comment:
you wouldn't put these in conditionals. the code doesn't execute until the element is clicked. you got some right ideas though on methods to use.